src/js/index.js

```diff
@@ -1,9 +1,11 @@
 import _ from "underscore";
 
+import {ELEMENT_BONDS} from "./element_bonds";
 import {PERIODIC_TABLE} from "./periodic_table";
 import {ELEMENT_COLORS} from "./element_colors";
 import {ELEMENT_VDW_RADII} from "./element_vdwRadii";
 
+export {ELEMENT_BONDS};
```

Comment:
Should we add a function executing some default logic for bonds that are not present above? Eg. Si-Ge as we spoke earlier

Reply:
The values should come from a reference. We should not assume any default values. If there is a bond for a pair of elements, it should be added to the table.

Reply:
I agree in principle, but in practice, this might be difficult to implement. So having an lets call it approximate, not default, way would be helpful.

Reply:
Added.